packages/libresprite-git/libresprite-git.pacscript

```diff
@@ -39,4 +44,8 @@ build() {
 install() {
 	cd build
 	sudo DESTDIR="$STOWDIR/$name" ninja install
+	cd ..
+	sudo install -Dm755 "libresprite.desktop" "$STOWDIR/$name/usr/share/applications/libresprite.desktop"
+	printf "Icon=/usr/share/pixmaps/libresprite.png" | sudo tee -a "$STOWDIR/$name/usr/share/applications/libresprite.desktop"
```

Comment:
```suggestion
	printf "Icon=libresprite" | sudo tee -a "$STOWDIR/$name/usr/share/applications/libresprite.desktop"
```

It should be just `libresprite` if it's in `pixmaps`, test it once before commiting though